travel-provider/TSP-30-RankTravelOptions/TSP-30-RankTravelOptionsRequest-RequestSchema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "TSP-30-RankTravelOptionsRequest Request Schema",
+  "version": "0.0.1",
+  "description": "Schema for ranking travel options request sent by CWT",
+  "properties": {
+    "returnId": {
+      "type": "integer",
+      "description": "Unique Id for this booking - generated by RMP."
+    },
+    "travelOptions": {
+      "type": "array",
+      "description": "A list of 1 to 9 travel options.",
+      "minItems": 1,
+      "maxItems": 9,
+      "items": {
+        "$ref": "#/definitions/travelOption"
+      }
+    },
+    "providerFeedback": {
+      "type": "object",
+      "description": "If fewer than 9 travel options are provided the reason must be specified.",
+      "properties": {
+        "limitedOptionsReason": {
+          "type": "string"
+        },
+        "limitedOptionsDescription": {
+          "type": "string"
+        }
+      }
+    }
+  },
+  "required": [
+    "returnId",
+    "travelOptions"
+  ],
+  "definitions": {
+    "travelOption": {
+      "type": "object",
+      "description": "A travel option that meets the requirements of the return case.",
+      "minItems": 1,
```

Comment:
This minItems constraint needs to move inside the flights property, not here. i.e. 

``` 
"properties": {
        "flights": {
          "type": "array",
           "minItems": 1,
          "items": {
            "$ref": "#/definitions/flight"
          }
        },
```